src/api/user/auth.js

```diff
@@ -63,10 +50,49 @@ exports.login = wrapper(async (req, res, next) => {
 })
 
 exports.confirm = wrapper(async (req, res, next) => {
-  const result = await jwtPromiseVer(req.query.token, 'registration')
+  const result = await verifyJwt(req.query.token, 'registration')
   const user = await User.findOne({ where: { email: result.email } })
   if (user.isVerified) { return next({ httpCode: 401, message: 'Already confirmed' }) }
 
   await user.update({ isVerified: true })
   res.json('Success confirmed')
 })
+
+// TODO need to destroy previous JWT Token ------------------
+exports.changePassword = wrapper(async (req, res, next) => {
+  const result = await verifyJwt(req.body.token, secret)
+  const user = await User.findOne({ where: { email: result.email } })
+  if (!user) return next({ httpCode: 404, message: 'No such user found' })
+
+  const successCompare = await bcrypt.compare(req.body.password, user.password)
+  if (!successCompare) return next({ httpCode: 401, message: 'Password doesn’t match' })
+
+  const hashedPassword = await bcrypt.hash(req.body.password_new, saltRounds)
+  await user.update({ password: hashedPassword })
+  res.json('Successfully changed!')
+})
+
+// TODO need to destroy previous JWT Token №2 ------------------
+exports.resetPassword = wrapper(async (req, res, next) => {
+  const result = await verifyJwt(req.query.token, 'forgot_password')
+  const user = await User.findOne({ where: { email: result.email } })
+
+  const hashedPassword = await bcrypt.hash(req.body.password, saltRounds)
+  await user.update({ password: hashedPassword })
+  res.json('Success reset!')
+})
+
+exports.askForgotPassword = wrapper(async (req, res, next) => {
+  const user = await User.findOne({ where: { email: req.body.email } })
+  if (!user) return next({ httpCode: 404, message: 'No such user found' })
+  const token = await jwt.sign({ email: req.body.email }, 'forgot_password', { expiresIn: '5m' })
+
+  const mailBody = {
+    token: token,
+    email: req.body.email,
+    route: url + '/api/users/reset-password',
```

Comment:
duplicated `url` variable I think you can take it to mail.js and here provide just '/api/users/reset-password'